release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3402,6 +3402,35 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
+// 4.14.46
+[id="ocp-4-14-46_{context}"]
+=== RHSA-2025:0840 - {product-title} 4.14.46 bug fix and security update
+
+Issued: 06 February 2025
+
+{product-title} release 4.14.46, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0840[RHSA-2025:0840] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:0842[RHSA-2025:0842] advisory.
```

Comment:
Seems like the advisories aren't live yet and report a 404. This isn't unexpected, but I wanted to note that here. :slightly_smiling_face: